packages/ui/src/slider.tsx

```
@@ -9,15 +9,22 @@ const Slider = React.forwardRef<
   React.ComponentPropsWithoutRef<typeof SliderPrimitive.Root> & {
     displayValue?: boolean;
   }
->(({ className, displayValue = false, ...props }, ref) => {
-  const [value, setValue] = React.useState(0);
+>(({ className, displayValue = false, disabled, ...props }, ref) => {
+  const [value, setValue] = React.useState(
+    (props.defaultValue && props.defaultValue[0]) || 0,
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)